webknossos-datastore/app/com/scalableminds/webknossos/datastore/services/mesh/MeshFileService.scala

```diff
@@ -106,7 +108,13 @@ class MeshFileService @Inject()(hdf5MeshFileService: Hdf5MeshFileService,
           } yield MeshFileInfo(meshFileName, mappingName, formatVersion)
         }
         // Only return successes, we don’t want a malformed file breaking the list request.
-        .map(_.flatten))
+        .map { boxes: Seq[Box[MeshFileInfo]] =>
```

Comment:
aren't there brackets needed here? a la
```suggestion
.map { boxes: Seq[Box[MeshFileInfo]] =>
          {
            boxes.filter(_.isEmpty).foreach { emptyBox =>
              logger.warn(s"Failed to list a mesh file for $dataSourceId: $emptyBox")
            }
            boxes.flatten
          }
        }
```

Because when I enter this, the formatter does not remove the bracket :thinking:

But on the contrary the compile seems to detect the `boxes.flatten` as defined.. Thus, your code might also be fine